### PR TITLE
Removing php 51 to 72 on travis

### PR DESCRIPTION
**Pull request by tom-christensen-mayden**

### CODE REVIEW

File: .phpunit.result.cache [hunk 1]
```diff
@@ -0,0 +1 @@
..."Spipu\\Html2Pdf\\Tests\\SvgDrawerTest::testTransform with data set #20":0,"Spipu\\Html2Pdf\\Tests\\Tag\\DivTest::testNoBreak":0.014,"Spipu\\Html2Pdf\\Tests\\Tag\\DivTest::testBreakBefore":0.012,"Spipu\\Html2Pdf\\Tests\\Tag\\DivTest::testBreakAfter":0.012,"Spipu\\Html2Pdf\\Tests\\Tag\\DivTest::testBreakBeforeAndAfter":0.013,"Spipu\\Html2Pdf\\Tests\\Tag\\MustHaveTagsTest::testOk":0.013,"Spipu\\Html2Pdf\\Tests\\Tag\\MustHaveTagsTest::testNotEmptyThead":0.005,"Spipu\\Html2Pdf\\Tests\\Tag\\MustHaveTagsTest::testNotEmptyTfoot":0.004,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\CircleErrorTest::testCase":0.008,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\EllipseErrorTest::testCase":0.008,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\GErrorTest::testCase":0.008,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\LineErrorTest::testCase":0.009,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\PathErrorTest::testCase":0.009,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\PathInvalidTest::testCase":0.008,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\PolygonErrorTest::testCase":0.007,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\PolylineErrorTest::testCase":0.009,"Spipu\\Html2Pdf\\Tests\\Tag\\Svg\\RectErrorTest::testCase":0.01,"Spipu\\Html2Pdf\\Tests\\Tag\\TdTooLongTest::testCase":0.016}}
```
Comment (elsbury13) [inline .phpunit.result.cache]: Do we need this file?

File: src/Tests/Exception/LongSentenceExceptionTest.php [hunk 2]
```diff
@@ -35,8 +36,7 @@ public function testBug()
         $html = '<page backleft="0" backright="200mm"style="font-size: 1mm">'.$bigSentence.'</page>';
 
         $object = $this->getObject();
-        $object->setSentenceMaxLines(100);
-        $object->writeHTML($html);
+        $object->setSentenceMaxLines(100);$object->writeHTML($html);
```
Comment (Geoff10) [inline src/Tests/Exception/LongSentenceExceptionTest.php]: Missing newline?

File: src/Tests/Html2PdfTest.php [hunk 3]
```diff
@@ -2,13 +2,22 @@
...
  * Class Html2PdfTest
  */
 class Html2PdfTest extends AbstractTest
 {
     public function testExtensionTag()
     {
+        /**
+         * Ignore deprecation errors as this causes this test to fail on deprecated toString.
+         * Later versions of phpunit allow this to be set in the XML file, but this isn't
+         * available until phpunit v6.2.
```
Comment (Geoff10) [inline src/Tests/Html2PdfTest.php]: I thought we are using PHPUnit 8 now?

File: src/Tests/Html2PdfTest.php [hunk 4]
```diff
@@ -2,13 +2,22 @@
```
Comment (Geoff10) [inline src/Tests/Html2PdfTest.php:2]: It looks to me like there are some tests in here that don't have assertions?

**Review by Geoff10 — CHANGES_REQUESTED** [review]
Just a couple of questions.

File: src/Tests/Html2PdfTest.php (hunk 4 above)
Comment (tom-christensen-mayden) [reply src/Tests/Html2PdfTest.php:2]: They don't assert anything but they do expect exceptions? What do we do in this case - they seem like poorly written tests with the testSecurityGood test not asserting anything before I added assertTrue(true) 😂

File: src/Tests/Html2PdfTest.php (hunk 3 above)
Comment (tom-christensen-mayden) [reply src/Tests/Html2PdfTest.php]: e44d456

File: src/Tests/Exception/LongSentenceExceptionTest.php (hunk 2 above)
Comment (tom-christensen-mayden) [reply src/Tests/Exception/LongSentenceExceptionTest.php]: 1060746

Comment (tom-christensen-mayden) [reply src/Tests/Exception/LongSentenceExceptionTest.php]: 9176b2a

File: src/Tests/Html2PdfTest.php (hunk 3 above)
Comment (tom-christensen-mayden) [reply src/Tests/Html2PdfTest.php]: 5b2e8cf

File: src/Tests/Html2PdfTest.php (hunk 4 above)
Comment (Geoff10) [reply src/Tests/Html2PdfTest.php:2]: Fair enough.

**Review by Geoff10 — APPROVED** [review]

**Review by elsbury13 — APPROVED** [review]

**Review by kyam-mayden — APPROVED** [review]

**Pull request merged**